packages/data-point/lib/reducer-types/factory.js

```diff
@@ -52,6 +52,10 @@ function normalizeInput (source) {
  * @return {reducer}
  */
 function createReducer (source, options = {}) {
+  if (isReducer(source)) {
+    return source
+  }
+
```

Comment:
now that `createReducer` is part of the API, we want to make sure that we don't get errors when doing something like this:

```js
const reducer = DataPoint.createReducer('$a')

dataPoint.resolve(reducer, { a: 1 })

```

Reply:
Nice!